lib/spreadsheet_architect/class_methods/xlsx.rb

```diff
@@ -43,7 +43,7 @@ def to_axlsx_package(opts={}, package=nil)
               end
             end
 
-            sheet.add_row header_row, style: header_style_index
+            sheet.add_row header_row, style: header_style_index, escape_formulas: true
```

Comment:
We shouldnt force the value to true here for the header row, it should use the option just the same as any other row

Reply:
That was my initial thought as well, but then I went back and forth with it a bit. The data passed to headers pretty much never is the same as what you would pass to the data rows, so just assuming the same preference would apply here actually feels a bit far-fetched, especially if using the array of booleans option.

If you think of the use cases, you could argue that people would quite rarely pass user input into a header row at all, but similarly you could say that it's a rare use case to want a formula in a header. That's why I ended siding with secure by default and went with this. Anyway, fine by me to go either way here.

Reply:
Yes please change it. The extra 1 header cell thats unsafe is totally fine in my mind. I dont want to restrict people in any way.